.github/workflows/node.js.yml

```diff
@@ -27,3 +27,25 @@ jobs:
         npm run build
         npm run fmt:check
         npm run test:ci
+    - id: coverage
+      run: |
+        value=$(npm run test:ci | grep -oE '^│\s*total\s+\│\s*([0-9.]+)%' | grep -oE '[0-9]+(\.[0-9]+)?%' | cut -d'.' -f1)
+        echo "coverage=$value" >> $GITHUB_OUTPUT
+
+    - name: Update README
+      run: |
+        "${GITHUB_WORKSPACE}/.github/coverage.sh"
+
+    - name: Commit changes
```

Comment:
i have the feeling that this won't work.
I think you have to:
- do the checkout using massabot account like [here](https://github.com/massalabs/massa-as-sdk/blob/main/.github/workflows/powered-by.yml#L16).
- set write permission like [here](https://github.com/massalabs/massa-as-sdk/blob/main/.github/workflows/powered-by.yml#L11).